scripts/test-eddsa-in-certificate-verify.py

```diff
@@ -0,0 +1,385 @@
+# Author: Hubert Kario, (c) 2021
+# Released under Gnu GPL v2.0, see LICENSE file for details
+"""Test for EdDSA support in Certificate Verify"""
```

Comment:
Don't we also need to test here that fuzzed or empty signatures in CertificateVerify are correctly rejected with decrypt error?